test/events/domains/proxy_domains_changed_event_test.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Domains::ProxyDomainsChangedEventTest < ActiveSupport::TestCase
+  test 'It can be deserialized when the Proxy is deleted' do
```

Comment:
I made it with TDD, this test would fail on master:
![image](https://user-images.githubusercontent.com/11318903/61461011-2abd6f80-a970-11e9-8862-851772c5f654.png)

But it passes here with the changes done 😉